Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'relaton-cli'
```

Comment:
We don't need the relaton-cli if we call the `RelatonIana::DataFetcher.fetch` directly.